daphne/src/roles_test.rs

```diff
@@ -1286,3 +1322,24 @@ async fn e2e() {
         .query_for_current_batch_window(t.now);
     t.run_col_job(task_id, &query).await.unwrap();
 }
+
```

Comment:
Reviewer note: There are a few more test cases we would like to add. They're on my TODO list -- right now I'm prioritizing getting to the point where we can start interop testing with others. That said, we could add them as part of this PR or document them in an issue.